### DISCUSSION TITLE
Planned Webinterface?

### DISCUSSION THREAD

**Discussion by ChR-iSz** [OvenMediaLabs/OvenMediaEngine#1102] [Ideas]
Hi, iam a web-developer expert.

Planned AirenSoft create a webinterface to configure OME?

I see a Beta API in v1. I think it is a good starting point to create client-side Application.

How much interest would there be in a web application?

**Comment by umeshshende** [comment]
You mean interface which will use OME API to show data and manage data such as server statistics,recording start/stop etc

**Comment by ChR-iSz** [comment]
Yes, exactly...

**Comment by getroot** [comment]
I know that some people are developing and using Web Console with OME API. If you develop the Web Console and release it as an open source, many people will like it.

**Comment by naanlizard** [comment]
Is there a git repo with work already?

**Comment by getroot** [comment]
@naanlizard AFAIK there is no public repo yet.

**Comment by gregfr** [comment]
The commercial version of the software provides a "web console for monitoring" (https://airensoft.com/enterprise.html)
